cmd/build.go

```diff
@@ -178,6 +178,29 @@ func subcommandBuild(targetOS string, packagingTask packaging.Task) {
 	assertHoverInitialized()
 	packagingTask.AssertInitialized()
 
+	if !buildSkipFlutterBuildBundle {
+		cleanBuildOutputsDir(targetOS)
+		buildFlutterBundle(targetOS)
+	}
+	if buildDocker {
+		var buildFlags []string
+		buildFlags = append(buildFlags, commonFlags()...)
+		buildFlags = append(buildFlags, "--skip-flutter-build-bundle")
+		buildFlags = append(buildFlags, "--skip-engine-download")
```

Comment:
These also apply to run.. Since it is possible to `hover run --docker`...

Reply:
Why were these moved anyway? I'm probably missing the reason so I don't understand yet how to approach this issue